app/views/course/statistics/_table.html.slim

```diff
@@ -0,0 +1,16 @@
+table.table.table-hover
+  tr
+    th = t('.serial_number')
+    th = CourseUser.human_attribute_name(:name)
+    th = t('.tutor')
+    th = Course::Level.model_name.human
+    th = t('.experience_points')
+
+  - students.each_with_index do |student, index|
+   = content_tag_for(:tr, student)
+     td = index + 1
+     td = link_to_user(student)
+     td = @service.tutors_of(student).map(&:name).join(', ')
```

Comment:
@kxmbrian As we discussed, might be good to just use a helper here. But think you can address this later, I will merge this first for waikay's course to use.